### PR TITLE
Changes to make the repo session-aware

### PR DESCRIPTION
**Pull request by Rahul-Nahar-1992**
Adding the ability to proxy requests via Session API in case sessionId was provided instead of an accessToken.

### CODE REVIEW

**Review by lholota — APPROVED** [review]

**Review by thoean — CHANGES_REQUESTED** [review]
Can you undo the formatting changes (whitespaces added where there weren't). If we want to change the formatting in this repository, we should do that as a separate MR.

You'll also need to increase the version in `package.json`, not only in the readme file.

**Comment by Rahul-Nahar-1992** [comment]
Hey @thoean, I have undone the formatting changes!

The `package.json` is already updated with the increased version (1.1.14).

**Comment by thoean** [comment]
Great, thanks. Ivan and Igor will take it from there who know the inner details of this change better.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.14] - 2021.05.19
```
Comment (istanishev) [inline CHANGELOG.md:7]: Likely this should be 1.2.0 as it is adding a new feature.

**Review by istanishev — APPROVED** [review]

**Pull request merged**